Bio/PDB/QCPSuperimposer/__init__.py

```diff
@@ -138,55 +138,5 @@ def get_rms(self):
 
 
 if __name__ == "__main__":
-    from datetime import datetime
-
-    # start with two coordinate sets (Nx3 arrays - float)
-
-    x = array([[-2.803, -15.373, 24.556],
-               [0.893, -16.062, 25.147],
-               [1.368, -12.371, 25.885],
-               [-1.651, -12.153, 28.177],
-               [-0.440, -15.218, 30.068],
-               [2.551, -13.273, 31.372],
-               [0.105, -11.330, 33.567]], 'f')
-
-    y = array([[-14.739, -18.673, 15.040],
-               [-12.473, -15.810, 16.074],
-               [-14.802, -13.307, 14.408],
-               [-17.782, -14.852, 16.171],
-               [-16.124, -14.617, 19.584],
-               [-15.029, -11.037, 18.902],
-               [-18.577, -10.001, 17.996]], 'f')
-
-    t0 = datetime.now()
-    for loop in range(0, 10000):
-        # start!
-        sup = QCPSuperimposer()
-
-        # set the coords
-        # y will be rotated and translated on x
-        sup.set(x, y)
-
-        # do the lsq fit
-        sup.run()
-
-        # get the rmsd
-        rms = sup.get_rms()
-
-        # get rotation (right multiplying!) and the translation
-        rot, tran = sup.get_rotran()
-
-        # rotate y on x
-        y_on_x1 = dot(y, rot) + tran
-
-        # same thing
-        y_on_x2 = sup.get_transformed()
-    t1 = datetime.now()
-    dif = t1 - t0
-    print("process wall time (msec): %d" % (dif.total_seconds() * 1000))
-
-    print(y_on_x1)
-    print("")
-    print(y_on_x2)
-    print("")
-    print("%.2f" % rms)
+    #moved test to Tests/test_QCPSuperimposer.py
+    # 2017 by Maximilian Greil
```

Comment:
You can remove all of this if statement (and in fact right now it is invalid syntax unless unless you add a ``pass`` command)